src/main/resources/static/js/ranking.js

```diff
@@ -1,3 +1,20 @@
+// Apply animation to each table row with delay
+$(window).on('load', function(){
+    var time = 0;
+    var delay = 20;
+    /* If we have more than x elements in the list, we just animate them without delay
+     * As with delay the displaying of the table rows could take for every.
+     */
+    var i = 0;
+    var stopDelayAfter = 20;
+    $('tr').each(function(){
+        var tr = this;
+        setTimeout(function(){$(tr).css('animation-play-state', 'running')}, time);
+        i++;
+        if (i < stopDelayAfter){time += delay;}
```

Comment:
Can you spread the if body over multiple lines instead of inlining it?
I think the intellj formatter `Ctrl + Alt + L` should take care of this and other formatting issues I found.

Reply:
I think its harder to read that way, as JS has a lot of parenthesis and I see a lot of other people coding that way.

Reply:
Specialy functions are used as parameter in JS and its a lot harder reading them if you formatting the code like you should do in Java in my opinion

Reply:
The way you formatted the code it's hard to see what belongs to what

Reply:
I changed it, maybe we need to discuss how we want to format JS and CSS next time. Specialy CSS Code gets a lot longer with our normal formatting guidelines